utils/create-quickstarts-snapshot.ts

```diff
@@ -0,0 +1,54 @@
+// To run (from the `utils/` directory):
+// npx ts-node create-quickstarts-snapshot.ts
+
+import * as fs from 'fs';
+import * as path from 'path';
+import {
+  FilePathAndContents,
+  findMainQuickstartConfigFiles,
+  readQuickstartFile,
+} from './helpers';
+import { buildMutationVariables } from './create_validate_pr_quickstarts';
+import { QuickstartConfig } from './types/QuickstartConfig';
+import { QuickstartMutationVariable } from './types/QuickstartMutationVariable';
+
+type QuickstartFileAndConfig = FilePathAndContents<QuickstartConfig>;
+
+/**
+ * Saves a JSON file for a quickstart, given the GraphQL Mutation variables.
+ * Files are saved in a top-level `snapshots/` directory.
+ *
+ * @example snapshots/battlesnake-12345.json
+ *
+ * @param variable - GraphQL Mutation variable for a quickstart
+ */
+const saveQuickstartMutationVariable = (
+  variable: QuickstartMutationVariable
+) => {
+  const { id, quickstartMetadata } = variable;
+  const { slug } = quickstartMetadata;
+
+  const filename = `${slug}-${id}.json`;
+  const filepath = path.join(__dirname, '..', 'snapshots', filename);
+
+  fs.writeFileSync(filepath, JSON.stringify(quickstartMetadata, null, 2));
```

Comment:
This fails if `snapshots/` isn't already created - i dont think we're planning on _keeping_ this script long term so might not be an issue